### ISSUE TITLE
MacOS 10.14 Support inquiry

### ISSUE THREAD

**Issue by maxmax2012** [dado3212/spaces-renamer#25]
**Describe the bug**
Hey, it just doesnt work on the new MacOs 10.14 Betas

Do you think you can add support, or let me know how to do so, if it's swift related, I can probably bang it myself.

The app seems super useful!!!!
Hope this can be a standalone!! (I'd like to help)

**Versions**
1. MySIMBL version - 0.7.1
2. Spaces Renamer version -lastest one 141-1 I believe

**Comment by codycodes** [comment]
Yeah I'm going to just post here on this issue and say that I'm still facing this issue. Keep opening it up and hoping it'll work 😅.

**Comment by dado3212** [comment]
Hey, I'm just going to close this, given that it's a repeat of #23, and that appears to be solved.  Make sure that SIP is disabled, that your mySIMBL version and spaces-renamer versions are the most recent ones, and you should be good to go.  If you can confirm that all of that's done, and it's still not working, open a new issue.

**Comment by ctrueden** [comment]
I was able to get Spaces Renamer working again on Mojave by doing the following:

* Disabled system integrity protection (as of Mojave, it has to _remain off_).
* Upgraded to latest mySIMBL (version 0.7.2 as of this writing).
* Upgraded to latest Spaces Renamer (version 1.5.1 as of this writing), dragging the plugin onto mySIMBL, and the application into `/Applications`, overwriting the old versions.
* Visited the mySIMBL "System Info" tab and clicked "Toggle AMFI" to disable Apple Mobile File Integrity.
* Rebooted.

Without disabling AMFI, the plugin was not working on my system anymore. Once I disabled AMFI and rebooted, all was well again.

Thanks to @dado3212 for this awesome plugin—it's saving me a lot of stress.
